dev/dll/XamlControlsResources.cpp

```diff
@@ -58,10 +59,11 @@ void XamlControlsResources::UpdateSource()
 
             const bool isInFrameworkPackage = SharedHelpers::IsInFrameworkPackage();
             const bool isInCBSPackage = SharedHelpers::IsInCBSPackage();
+            const bool areVisualUpdateResourcesAvailable = SharedHelpers::AreVisualUpdateAPIsAvailable() && assumeVisualUpdateResourcesAvailable;
 
             hstring compactPrefix = useCompactResources ? L"compact_" : L"";
             hstring packagePrefix = L"ms-appx:///" MUXCONTROLSROOT_NAMESPACE_STR "/Themes/";
-            hstring postfix = useNewVisual ? L"themeresources.xaml" : L"themeresources_2dot5.xaml";
+            hstring postfix = useNewVisual ? (areVisualUpdateResourcesAvailable ? L"themeresources_vu.xaml" : L"themeresources.xaml") : L"themeresources_2dot5.xaml";
```

Comment:
It's a little weird here. I don't expect _vu here

Reply:
because themeresource.xaml includes latest visual and themeresources_2dot5.xaml includes old visual

Reply:
...and themeresource_vu.xaml includes references to recent 21H1 OS build APIs.